### PR TITLE
[Merged by Bors] - fix: correct deprecation suggestions for algebra classes

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Init/Algebra/Classes.lean [hunk 1]
```diff
@@ -72,7 +72,7 @@ class IsSymmOp (α : Sort u) (β : Sort v) (op : α → α → β) : Prop where
 #align is_symm_op IsSymmOp
 
 /-- A commutative binary operation. -/
-@[deprecated IsCommutative]
+@[deprecated Std.Commutative] -- 2024-02-02
 abbrev IsCommutative (α : Sort u) (op : α → α → α) := Std.Commutative op
 #align is_commutative IsCommutative
```
Comment (adomani) [inline Mathlib/Init/Algebra/Classes.lean:77]:
```suggestion
#align is_commutative Std.Commutative
```
not sure, I'm on mobile.

File: Mathlib/Init/Algebra/Classes.lean [hunk 2]
```diff
@@ -81,16 +81,16 @@ instance (priority := 100) isSymmOp_of_isCommutative (α : Sort u) (op : α →
 #align is_symm_op_of_is_commutative isSymmOp_of_isCommutative
 
 /-- An associative binary operation. -/
-@[deprecated IsAssociative]
+@[deprecated Std.Associative] -- 2024-02-02
 abbrev IsAssociative (α : Sort u) (op : α → α → α) := Std.Associative op
```
Comment (Ruben-VandeVelde) [inline Mathlib/Init/Algebra/Classes.lean:85]: This got lost in #10176:
```suggestion
abbrev IsAssociative (α : Sort u) (op : α → α → α) := Std.Associative op
#align is_associative Std.Associative
```

**Comment by adomani** [comment]
Thanks!

maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**